packages/adapter-vercel/index.js

```diff
@@ -11,13 +11,26 @@ const name = '@sveltejs/adapter-vercel';
 const DEFAULT_FUNCTION_NAME = 'fn';
 
 const get_default_runtime = () => {
-	const major = process.version.slice(1).split('.')[0];
-	if (major === '18') return 'nodejs18.x';
-	if (major === '20') return 'nodejs20.x';
+	const major = Number(process.version.slice(1).split('.')[0]);
+
+	// If we're building on Vercel, we know that the version will be fine. Also means
+	// we're not on the hook for updating the adapter every time a new Node version is
+	// added to Vercel.
```

Comment:
```suggestion
	// If we're building on Vercel, we know that the version will be fine because Vercel
	// provides Node (and it's impossible that Vercel provides somthing it doesn't support).
	// Also means we're not on the hook for updating the adapter every time a new Node
	// version is added to Vercel.
```